### ISSUE TITLE
PyPI import missing utils

### ISSUE THREAD

**Issue by Liresol** [rhgrant10/acopy#19]
* ACOpy version: 0.6.1
* Python version: 3.7.1, installed using homebrew
* Operating System: macOS Mojave (10.14)

### Description

Currently (or as an isolated problem on my system), installing the package with pip does not also install the utils folder, which is required for the package to properly function. 

My makeshift fix was simply moving the utils folder from this project into the site-packages folder where the package was stored. So far, this has solved/suppressed all errors, so it seems like the imported package is simply missing utils.

If the problem is unsolvable/unimportant/known, then this issue can be closed immediately, but I just wanted to bring it up in case it inconvenienced anyone.

### What I Did

In terminal:
```
pip install acopy
```
In the python interpreter:
```
>>> import acopy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/acopy/__init__.py", line 2, in <module>
    from .ant import Ant  # noqa: F401
  File "/usr/local/lib/python3.7/site-packages/acopy/ant.py", line 7, in <module>
    from . import utils
ImportError: cannot import name 'utils' from 'acopy' (/usr/local/lib/python3.7/site-packages/acopy/__init__.py)
```
Moving ```acopy/utils``` from this repository into ```/usr/local/lib/python3.7/site-packages/acopy``` causes the above to give no error.

**Comment by rhgrant10** [comment]
My apologies, it seems I forgot to circle back and communicate the fix was out. I hope you saw the patch bump and have been using v0.6.2.

**Comment by rhgrant10** [comment]
Looks like the fix didn't fix enough. Submitting a real fix for this momentarily.

**Comment by rhgrant10** [comment]
This is verified fixed in v0.6.3:
```shell
$ python
Python 3.7.2 (default, Dec 27 2018, 07:35:52)
[Clang 10.0.0 (clang-1000.11.45.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```
```python
>>> import acopy
>>> acopy.utils
<module 'acopy.utils' from '/Users/rhgrant10/code/acopy/acopy/utils/__init__.py'>
>>> acopy.__version__
'0.6.3'
```

**Comment by Synthetica9** [comment]
This still doesn't seem to work (on my machine):

```
  File "main.py", line 6, in <module>
    import acopy
  File "/home/synthetica/.local/share/virtualenvs/portmantout-NI50_7Zr/lib/python3.7/site-packages/acopy/__init__.py", line 2, in <module>
    from .ant import Ant  # noqa: F401
  File "/home/synthetica/.local/share/virtualenvs/portmantout-NI50_7Zr/lib/python3.7/site-packages/acopy/ant.py", line 7, in <module>
    from .utils import positive
ModuleNotFoundError: No module named 'acopy.utils'
```

When I install everything with pipenv.

<details>
  <summary>`Pipfile.lock` for completeness</summary>
  
```
{
    "_meta": {
        "hash": {
            "sha256": "24b339beef93e96ae9cdfaef339666ebf42fad8fc95d3fdd6fe8cadb6a79f5a4"
        },
        "pipfile-spec": 6,
        "requires": {
            "python_version": "3.7"
        },
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {
        "acopy": {
            "hashes": [
                "sha256:e652f7dce9bc51026dc65f35ffe3109ac9962a6590c3d5ec11be51b13a5ba651",
                "sha256:ff3192b50079939c7b1331d5a5b14ddd23680b5cc85a272018f08b1882fb755d"
            ],
            "index": "pypi",
            "version": "==0.6.3"
        },
        "click": {
            "hashes": [
                "sha256:29f99fc6125fbc931b758dc053b3114e55c77a6e4c6c3a2674a2dc986016381d",
                "sha256:f15516df478d5a56180fbf80e68f206010e6d160fc39fa508b65e035fd75130b"
            ],
            "version": "==6.7"
        },
        "decorator": {
            "hashes": [
                "sha256:86156361c50488b84a3f148056ea716ca587df2f0de1d34750d35c21312725de",
                "sha256:f069f3a01830ca754ba5258fde2278454a0b5b79e0d7f5c13b3b97e57d4acff6"
            ],
            "version": "==4.4.0"
        },
        "marisa-trie": {
            "hashes": [
                "sha256:4419abb6b603c97e863fad994abe57ed247fb12491f4bbacb2d762bd2e8958b6",
                "sha256:c73bc25d868e8c4ea7aa7f1e19892db07bba2463351269b05340ccfa06eb2baf"
            ],
            "index": "pypi",
            "version": "==0.7.5"
        },
        "networkx": {
            "hashes": [
                "sha256:64272ca418972b70a196cb15d9c85a5a6041f09a2f32e0d30c0255f25d458bb1"
            ],
            "index": "pypi",
            "version": "==2.1"
        },
        "tsplib95": {
            "hashes": [
                "sha256:a7672f7b9d1ae06c80639a6585e71c79c0459a7dc0eb9babf51e302fe3552d51",
                "sha256:e0e6905a35f29234de496a874905413b9c9ce437a8d572e95351cc255c16be83"
            ],
            "version": "==0.3.2"
        }
    },
    "develop": {}
}
```
</details>

**Comment by rhgrant10** [comment]
@Synthetica9 thanks for including the pipfile.lock - I'm able to reproduce. Working on a fix.

**Comment by rhgrant10** [comment]
@Synthetica9 okay I believe I have found the true source of this issue.

The invocation of `setuptools.find_packages` in the `setup.py` actually specified just the root package, thereby implicitly excluding all other packages (even nested ones) from the packaging process. When I performed local testing and the imports seemed in working order, it was only because of the way pip does local installs when passing the `--editable` flag. (I actually had to install the local copy without using editable _and_ be in another directory before I could reproduce the error without having to install it from PyPI.)

[v0.6.4 on PyPI](https://pypi.org/project/acopy/0.6.4/)

Three fixes is a charm, right? Anyway, thanks for your patience.
